### PR TITLE
Format Readme + Update doc link to ours

### PR DESCRIPTION
**Pull request by Mkassabov**
### What this does
- [x] formats readme according to markdown rules for code
- [x] changes the specification link to our repo (it doesn't seem like we open a PR in their repo for it so I assume its a custom change?)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![Version](https://img.shields.io/npm/v/@investingwolf/alpaca-broker-api)](https://www.npmjs.com/package/@investingwolf/alpaca-broker-api)
 
-Node.js client for the Alpaca Broker API generated from the specs provided [here](https://github.com/alpacahq/bkdocs).
+Node.js client for the Alpaca Broker API generated from the specs provided [here](https://github.com/investingwolf/bkdocs).
```
Comment (jacobparish) [inline README.md:5]: maybe `[our forked version](link) of the specs provided [here](other link)`

File: README.md [hunk 2]
```diff
@@ -19,12 +19,15 @@ npm install @investingwolf/alpaca-broker-api
 You will need an Alpaca API key and API secret. You can then create an Alpaca client as follows:
 
 ```js
-const {AlpacaApi, AlpacaEnvironments} = require('@investingwolf/alpaca-broker-api');
+const {
+  AlpacaApi,
+  AlpacaEnvironments,
+} = require("@investingwolf/alpaca-broker-api");
```
Comment (jacobparish) [inline README.md:26]: Where is this format coming from? I don't think this repo even has an eslint config. should we add one?

**Comment by jacobparish** [comment]
>  changes the specification link to our repo (it doesn't seem like we open a PR in their repo for it so I assume its a custom change?)

I was planning on opening a PR in their repo at some point. they just don't have all the endpoints yet so I added a few

File: README.md (hunk 2 above)
Comment (Mkassabov) [reply README.md:26]: probably my default lint config, we should probably just use the one we use in our BE repo

**Pull request closed without merging**